README.md

```diff
@@ -10,24 +10,50 @@ Inmanta Documentation on [Read the Docs](https://inmanta.com/resources/docs/)
 
 ## Features
 
-For now, just basic Syntax Coloring
 
-![Screenshot](images/screenshot.png)
+### Syntax coloring
+The extension makes your life easier by coloring syntax keywords or the Inmanta language.
 
-## Alpha Features
+![Syntax coloring screenshot](images/screenshot-syntax-coloring.png)
 
 ### Code navigation
+The extension allows you to navigate to the Definition of types used by using the shortcut `Ctrl + Click` when you hover the element.  And will show an overview of the Entity if you don't click.
 
-To enable ctrl-click for code navigation, take the following steps:
+![Navigation screenshot](images/screenshot-ctrl-click.png)
+
+### Compilation error reporting
+The extension will run a pre-compilation every time you save your file.  Making it easier for you to spot syntax and basic model errors.  When hovering the problematic bit, an explaination message is shown, and more details can be found in the **Output** panel (when selecting *Inmanta Server Language* in the expandable list on the right).
+
+![Error reporting screenshot](images/screenshot-error-reporting.png)
+
+### Model export
```

Comment:
purely informative: is this a build task? (i.e. if you press f5, does it export?)

Reply:
We would need at least to add a task to `.vscode/tasks.json` to be able to do that right?